<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>850</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="expInfoGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>741</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어_ac</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Experiments Infomation</string>
   </property>
   <widget class="QLabel" name="nameLabel2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>InTaekJung</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="test_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pre-TEST (경제)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="expCntLabel2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1회차</string>
    </property>
   </widget>
   <widget class="QLabel" name="name2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이름: </string>
    </property>
   </widget>
   <widget class="QLabel" name="expCnt2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>회차:</string>
    </property>
   </widget>
   <widget class="QLabel" name="birth_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>생년월일:</string>
    </property>
   </widget>
   <widget class="QLabel" name="birthLabel2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>901114</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TeInfoGBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>741</width>
     <height>761</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어_ac</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Test Infomation</string>
   </property>
   <widget class="QGroupBox" name="testGBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>611</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>문제지</string>
    </property>
    <widget class="QLabel" name="testLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>581</width>
       <height>621</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ansGBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>선택지</string>
    </property>
    <widget class="QRadioButton" name="ansRBtn1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ansRBtn5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="testCntLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>740</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>1 / 5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="teSubmitBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>680</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>제출하기</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="etcGBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>101</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>Etc.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
